### DISCUSSION TITLE
PiHole keep disconnecting

### DISCUSSION THREAD

**Discussion by rickninja765** [DesktopECHO/Pi-hole-for-Android#61] [Show and tell]
Whenever Router tries to reboot, it assigns new IP to each other devices. So Pihole gets new IP and internet is disconnected cause router has specific ip of older pihole ip. So had to enable new IP in Router. Anyway to assign static IP to pihole to resolve the issue.

**Comment by DesktopECHO** [comment]
You can set a static IP on your Android device, [instructions here](https://service.uoregon.edu/TDClient/KB/ArticleDet?ID=33742).
